### PR TITLE
[Feature] Chakra UI theme setup

### PR DESCRIPTION
**Pull request by OustanDing**
## Notion ticket link

[Ticket](https://www.notion.so/uwblueprintexecs/Configure-Chakra-UI-theme-b517ab3f4c9b4b559fc530254689fe28)

## Implementation description
* Configured theme typography, colours, shadows, spacing
* Configured default text styles
* Overrode default component styles and tested custom styles
* Installed fonts
* Small changes to existing frontend

## Checklist
- [x] My PR name is descriptive, is in imperative tense and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket

### CODE REVIEW

File: pages/login.tsx [hunk 1]
```diff
@@ -20,12 +20,30 @@ export default function Login() {
...
+          <Input
+            type="email"
+            placeholder="Your email"
+            value={email}
+            onChange={event => setEmail(event.target.value)}
+            onKeyUp={handleKeyUp}
+          />
+          <FormHelperText>Please enter an RCD email</FormHelperText>
+        </FormControl>
+        <Button onClick={signInWithEmail}>Log in</Button>
```
Comment (jeffreyzhang2001) [inline pages/login.tsx:46]: Can we refactor this slightly? I think we can wrap the entire thing up in a form like this, without having to write an onKeyUp fn to support enter key submit.

```
<form>
  <FormControl>
        ....
  </FormControl>
  <Button onClick={() => signInWithEmail}>Log in</Button>
</form>
```

**Review by jeffreyzhang2001 — CHANGES_REQUESTED** [review]

**Comment by jeffreyzhang2001** [comment]
Just one small nit. Otherwise, LGTM

**Review by jeffreyzhang2001 — APPROVED** [review]

File: pages/login.tsx [hunk 2]
```diff
@@ -12,38 +12,41 @@ export default function Login() {
...
-            onChange={event => setEmail(event.target.value)}
-            onKeyUp={handleKeyUp}
-          />
-          <FormHelperText>Please enter an RCD email</FormHelperText>
-        </FormControl>
-        <Button onClick={signInWithEmail}>Log in</Button>
+        <form onSubmit={handleSubmit}>
+          <FormControl>
+            <FormLabel>Login</FormLabel>
+            <Input
```
Comment (OustanDing) [inline pages/login.tsx:38]: Could we also disable the Input field while it's signing in?

**Review by OustanDing — COMMENTED** [review]
@jeffreyzhang2001 Changes look great. Just one small suggestion then we can ship it

**Pull request merged**